PersonalBlog/urls.py

```diff
@@ -2,10 +2,18 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path,include
+from rest_framework import routers
+from BlogApp.views import UserViewSet, BlogPostViewSet
+
+router = routers.DefaultRouter()
+router.register(r'users', UserViewSet)
+router.register(r'blogposts', BlogPostViewSet)
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/', include(router.urls)),
```

Comment:
Need to remove this code too :)

Reply:
I mean i have already implemented the API in my branch so you do not need to do anything for now :)